packages/opc-feedback/src/opc-feedback.scss

```
@@ -0,0 +1,108 @@
+@import '@one-platform/opc-styles';
+
+$LOCAL: feedback;
+
+$variables: ( 
+  //Add your variables here
+);
+
+:host {
+  --op-transition--default: 120ms ease-in-out;
+  --op-feedback__btn-primary: #b70202;
+  --op-feedback__bg-primary: #fceaeb;
+
+  position: relative;
+  z-index: 9000;
+}
+
+* {
+  outline-width: medium;
+  font-family: 'Red Hat Text', 'Open Sans', Arial, Helvetica, sans-serif;
+}
+
+.op-feedback__button {
```

@riginoommen This could be 
```suggestion
.op-feedback {
&__button {
```

---

Applied the suggested changes

---

Applied the suggested changes